super-cool-game/Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -1,72 +1,52 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
+﻿using UnityEngine;
 
 public class PlayerMovement : MonoBehaviour {
-    /* Handles the movement of the player.
-     * This script is always attached to the player's character game object,
-     * while PlayerInput is attached only when the character's owner is the local player.
-     */
-    
-    // These ones can be edited in the inspector
-    public float speed = 3.0f;
-    public float maxSpeed = 9.0f;
-    public float jumpHeight = 6.0f;
-    public float checkGroundDistance = 1.0f;
-    public LayerMask groundLayer;
+    /* Translates commands from PlayerController into movement */
 
     private Rigidbody2D rb;
-    private float moveX = 0;
+    private BoxCollider2D boxCollider;
 
-    // Use this for initialization
+    private float friction = 0.4f;
+    private float velX = 0;
```

Comment:
`private float velX = 0;`: use `public` instead of `private` for quick editor changes.

Reply:
I'd prefer private because by changing the values in the script i can change them for every object or prefab that inherits the script, otherwise I'd need to edit them everywhere manually. Not an issue now but it is in the multiplayer PR